js/app/src/window.ts

```diff
@@ -0,0 +1,109 @@
+// The MIT License (MIT)
```

Comment:
compatible license?
Binary should probably have a link, menu item, or button that provides license information and references to all of the copyright files.

Reply:
> Binary should probably have a link, menu item, or button that provides license information and references to all of the copyright files.

Good call, I agree.

It will be a large document... here is the print out of all of the licenses within all of the dependencies utilised:

[COPYING.txt](https://github.com/pymedphys/pymedphys/files/8973623/COPYING.txt)